bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -10,6 +10,9 @@
   {% if switch('firefox_new_better_browser_experiment', ['en-US']) %}
     {{ js_bundle('firefox_new_better_browser_experiment') }}
   {% endif %}
+  {% if switch('firefox_new_compare_safari_experiment', ['en-US']) %}
+    {{ js_bundle('firefox_new_compare_safari_experiment') }}
```

Comment:
I have concerns about these experiments colliding. In theory, even with multiple experiments running, they should each only take effect when the URL matches the experiment, but I suspect we might need to wrap add some other condition so it doesn't even load unless the URL is one we're testing... `if xv == 'safari'` or something?

Reply:
Maybe the better browser experiment has been running long enough?

Reply:
Possibly. Though we also know we'll need to have Safari, Edge, and Chrome pages running simultaneously, each with A and B variants, so we'll be in the same boat.